python/pacemaker/_cts/remote.py

```diff
@@ -114,81 +131,90 @@ def __init__(self, command, cp_command, silent=False):
         self._our_node = os.uname()[1].lower()
 
     def _fixcmd(self, cmd):
-        """ Perform shell escapes on certain characters in the input cmd string """
+        """Perform shell escapes on certain characters in the input cmd string"""
 
-        return re.sub("\'", "'\\''", cmd)
+        return re.sub("'", "'\\''", cmd)
 
     def _cmd(self, args):
-        """ Given a list of arguments, return the string that will be run on the
-            remote system
+        """Given a list of arguments, return the string that will be run on the
+        remote system
         """
 
         sysname = args[0]
         command = args[1]
 
-        if sysname is None or sysname.lower() == self._our_node or sysname == "localhost":
+        if (
+            sysname is None
+            or sysname.lower() == self._our_node
+            or sysname == "localhost"
+        ):
```

Comment:
Function candidate

Reply:
This can be shortened up a little bit (getting it down to 80 characters) by using a list instead.